### ISSUE TITLE
[question]: adding a service extension for IOS in Maui

### ISSUE THREAD

**Issue by LucaDuttoTD** [OneSignal/OneSignal-DotNet-SDK#126]
### How can we help?

Hello. I'm porting to MAUI an existing application. This application is (obviously!) using OneSignal for push notification.

I'm following the guide here: https://documentation.onesignal.com/docs/net-sdk-setup
but I'm stuck at 3: Adding A Service Extension
Is the documentation outdated? or am I missing something obvious? Because I'm unable to get it right: the documentation points to a MS page about Xamarin, trying to create a service extension anyway results in an empty folder...
Can you point me in the right direction for creating this Service Extension? Or: what happens if I skip this passage?

I think a clearer documentation would be very appreciated and useful for the community.

Thanks a lot

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by DaveBrask** [comment]
When I got to where you are I didn't want to make one.  It seems the extension is for changing the incoming message.  I didn't make one and it works just fine.

**Comment by nan-li** [comment]
Hi @LucaDuttoTD you are seeing the instructions on https://learn.microsoft.com/en-us/previous-versions/xamarin/ios/platform/user-notifications/enhanced-user-notifications?tabs=windows#implementing-a-service-extension does not actually work?

**Comment by mads195** [comment]
> Hi @LucaDuttoTD you are seeing the instructions on https://learn.microsoft.com/en-us/previous-versions/xamarin/ios/platform/user-notifications/enhanced-user-notifications?tabs=windows#implementing-a-service-extension does not actually work?

No - that specific step only works on Xamarin projects. Maui appears to not have Service Extensions natively. Happy to be corrected if I'm wrong.

**Comment by LucaDuttoTD** [comment]
> Hi @LucaDuttoTD you are seeing the instructions on https://learn.microsoft.com/en-us/previous-versions/xamarin/ios/platform/user-notifications/enhanced-user-notifications?tabs=windows#implementing-a-service-extension does not actually work?

Nope, it doesn't seem to work, as @mads195 noted it seems to be specific for Xamarin
